assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -378,9 +378,10 @@ export default function ContribAmounts(props: PropTypes) {
             onFocus={props.changeContribAmount}
             onInput={props.changeContribAmount}
             selected={attrs.selected}
-            placeholder={`Other amount (${props.currency.glyph})`}
+            placeholder="Other amount"
```

Comment:
I wish react used single quotes like the rest of our js

Reply:
It's mimicking HTML 🙃